packages/@okta/vuepress-theme-prose/assets/css/components/_multiTenancy.scss

```diff
@@ -0,0 +1,152 @@
+$blue: #163648;
```

Comment:
What's the motivation for this doc having bespoke CSS?

Reply:
jakubvul-okta - I used custom css for this page to include design elements that weren’t available in markdown. 

alexdahl-okta - Thanks for the feedback. You’re right it’s not a reusable component.  I could use some direction on where to organize the styles that I'm using. I'll set up 30 minute to go over that on Monday.  I’ll include Jakub as well.

Reply:
The latest commit updates the code based on the meetings and discussion concerning the page's css and css location.